app/controllers/concerns/vet360/writeable.rb

```diff
@@ -22,6 +22,22 @@ def write_to_vet360_and_render_transaction!(type, params, http_verb: 'post')
       render_new_transaction!(type, response)
     end
 
+    # Temporary method for debugging during UAT
+    #
+    def log_profile_data_to_sentry(response)
+      log_message_to_sentry(
+        'Profile controller bug',
+        :info,
+        {
+          controller: self.class.to_s,
+          response: response,
+          params: params,
+          user: @current_user.inspect
```

Comment:
@kreek - Question, I was under the impression that between the stripping that Sentry does, and our custom filtering/stripping done in:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/sentry/processor/pii_sanitizer.rb

That we were safe to log the user object….no?  

@jkassemi mentioned that perhaps the `.inspect` wouldn't be accounted for.  

Thoughts on switching this to `user: @current_user`, and shipping the rest of the PR, as is?

cc: @rlbaker

Reply:
I'm not sure the blacklist will cover the full set of attributes for the user object. Two options at this point:

1) give this a shot with a user object from the dev environment in a rails console, and see what gets filtered out in sentry. Then make those modifications where necessary. I'm pretty sure `first_name`, `last_name`, `middle_name` and `zip` aren't accounted for, for instance.
2) log the message without current user, and coordinate with EVSS on the requests they received around that time when the issues were encountered
3) coordinate with EVSS on setting up a header that can be used for tracing in their logs. Preference on `X-Request-ID`. We could use pass through the rails request identifier with a thread local for now, and potentially in the future if vets-api orchestrates multiple requests to partner services we could set up unique ids for this value in the future.
4) just make sure that all of these issues are deleted from sentry after the fact. Sentry isn't actually "forever" (apologies for not catching that the other day) - so as long as we're not logging to cloudwatch logs this isn't much of a problem. 

I lean pretty strongly toward #3 happening regardless (with EVSS && Vet360, though I believe Vet360 has provided some of this functionality based on earlier design docs). This can be used at multiple points in the support / debug response chain. cc @mattpointzxer0 for visibility on this as a request from EVSS.

Reply:
Update:  Since I initially created this, and the former, PR, we have made some discoveries with EVSS.  They have not released their email and phone endpoints to production at all.  This was the source of the issues.

That is being addressed on their side, and will be resolved next week.

That said, I am going to recommend that we go with #2, and just remove the user logging completely.  We do not need it for our side, and nor do they.

This will get the ball rolling, and if we need more logging based on their release to production, we can always add more.

@jkassemi @kreek @rlbaker - That work with you?

Reply:
@hpjaj Your strategy looks solid to me.

@jkassemi I 100% agree that point #3 would be extremely useful to move towards as a strategy for debugging these types of issues in the future. Is there a good spot to ticket that sort of work that maybe doesn't fall under a certain team designation at the moment?

Reply:
@rlbaker - https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10441

Reply:
@hpjaj - works for me!